TUTORIAL

```diff
@@ -808,7 +808,7 @@ test/Makefile.am:
 
 We need to add src/.libs/ to PATH environment variable
 before run cutter to find DLL generated under src/.libs/ on
-Cygwin:
+Cygwin, Darwin (including MacOS), and BSD Unix:
```

Comment:
It seems that this change describes wrong thing. Because macOS and BSD don't use DLL and PATH.

Could you use separated paragraph or something to describe this?

And could you use "macOS" instead of "MacOS" because "macOS" is the official form.